patches/0013-p256-armv8-optimizations

```diff
@@ -2429,20 +2457,460 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256-x86_64_test.cc
      !defined(OPENSSL_SMALL) && !defined(BORINGSSL_SHARED_LIBRARY)
  
  TEST(P256_X86_64Test, SelectW5) {
-@@ -98,6 +97,7 @@ TEST(P256_X86_64Test, SelectW7) {
-   CHECK_ABI(ecp_nistz256_select_w7, &val, table, 42);
+   // Fill a table with some garbage input.
+-  alignas(64) P256_POINT table[16];
++  __attribute__((aligned(64))) P256_POINT table[16];
+   for (size_t i = 0; i < 16; i++) {
+     OPENSSL_memset(table[i].X, 3 * i, sizeof(table[i].X));
+     OPENSSL_memset(table[i].Y, 3 * i + 1, sizeof(table[i].Y));
+@@ -71,7 +70,7 @@ TEST(P256_X86_64Test, SelectW5) {
+ 
+ TEST(P256_X86_64Test, SelectW7) {
+   // Fill a table with some garbage input.
+-  alignas(64) P256_POINT_AFFINE table[64];
++  __attribute__((aligned(64))) P256_POINT_AFFINE table[64];
+   for (size_t i = 0; i < 64; i++) {
+     OPENSSL_memset(table[i].X, 2 * i, sizeof(table[i].X));
+     OPENSSL_memset(table[i].Y, 2 * i + 1, sizeof(table[i].Y));
+@@ -99,11 +98,12 @@ TEST(P256_X86_64Test, SelectW7) {
  }
  
-+#if !defined(OPENSSL_AARCH64)
  TEST(P256_X86_64Test, BEEU) {
++#if !defined(OPENSSL_AARCH64)
    if ((OPENSSL_ia32cap_P[1] & (1 << 28)) == 0) {
      // No AVX support; cannot run the BEEU code.
-@@ -165,6 +165,8 @@ TEST(P256_X86_64Test, BEEU) {
+     return;
    }
- }
- 
+-
 +#endif
+   bssl::UniquePtr<EC_GROUP> group(
+       EC_GROUP_new_by_curve_name(NID_X9_62_prime256v1));
+   ASSERT_TRUE(group);
+Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/asm/p256_beeu-armv8-asm.pl
+===================================================================
+--- /dev/null
++++ aws-lc/third_party/boringssl/crypto/fipsmodule/ec/asm/p256_beeu-armv8-asm.pl
+@@ -0,0 +1,423 @@
++# Copyright (c) 2020, Amazon Inc.
```

Comment:
There should probably be an explicit license here.

Reply:
I changed it. Thanks.